environment.yaml

```diff
@@ -3,40 +3,40 @@ channels:
 - main
 dependencies:
 - autopep8=1.5.4
-- beautifulsoup4=4.9.1
+- beautifulsoup4=4.9.3
 - click=7.1.2
 - codecov=2.1.9
 - dill=0.3.2
 - flake8-isort=4.0.0
-- flake8=3.8.3
-- geckodriver=0.27.0
-- ipython=7.17.0
+- flake8=3.8.4
+- geckodriver=0.26.0
```

Comment:
@birdsarah do you have any idea why repining would have us switch to an older version of geckodriver?